solutions/dmoj-rank.mdx

```diff
@@ -121,4 +121,78 @@ for i in range(n):
 print(ans)
 ```
 </PySection>
+
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class rank {
+    static Map<Integer, Set<Integer>> graph = new HashMap<>();
+    // Key is winner and Value is loser
+    static int ans = 0;
+    static Set<Integer> visited = new HashSet<>();
+    static boolean cyclic = false;
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        StringTokenizer st = new StringTokenizer(br.readLine());
+        int people = Integer.parseInt(st.nextToken());
+        int games = Integer.parseInt(st.nextToken());
+
+        for (int x = 0; x < games; x++) {
+            st = new StringTokenizer(br.readLine());
+            int p1 = Integer.parseInt(st.nextToken()); // Person 1
```

Comment:
two spaces after code for inline comment

Reply:
Added two spaces for inline comments